src/renderer/components/ft-input/ft-input.css

```diff
@@ -174,6 +177,8 @@
 /*   color: white; */
 }
 
-.showClearTextButton .ft-input {
-  padding-left: 2em;
+.clearInputTextButton.visible ~ .ft-input {
+  /* 36px width of clearInputTextButton plus 6px of margin */
+  padding-left: calc(36px + 6px);
+  transition: padding 0.1s;
```

Comment:
It's an interesting one, I found that making it slower is worse as the text continues moving _as you type_. Potentially there should be no transition at all here and we should instead keep the text static. Maybe "disable" the clear input button the same way the action button is disabled?

Reply:
@absidue I pushed the thought I had above, works in a much more straightforward but maybe less aesthetically appealing way now

Reply:
This looks better than having a blank space there.

If you transition the padding, it might look better than having it appear instantly. Also it's probably a good idea to have the clear button and padding visible regardless of the focus, when the search box has a value, that way it can be cleared without needing to focus the search box first.

Reply:
I think a transition there is unnecessary, syncing the padding extending and the icon showing would be non-trivial too. Good catch with it not being visible when out of focus, fixed that now.

Reply:
looks good now, could you also please fix the typo in the class name in the latest commit :) ? After that i'm happy to approve it.

Reply:
Old English word vibisle, never heard of it? Fixed ;)